hist/hist/src/THStack.cxx

```diff
@@ -234,7 +234,7 @@ THStack::THStack(TH1* hist, Option_t *axis /*="x"*/,
       }
       TAxis* haxis= useX ? hist->GetYaxis() : hist->GetXaxis();
       if (!haxis) {
-         Warning("HStack","Histogram axis is NULL");
+         Warning("THStack","Histogram axis is NULL");
```

Comment:
Why not adding in the message if the x or y axis is NULL

Reply:
Looking at the code it is not so obvious to know straight away which axis it is ... unless we had more code which will complexify the code for nothing. I changed these lines because the "T" was missing but that's not the purpose of this PR.